playground/deepspeed_config_s2.json

```diff
@@ -2,11 +2,23 @@
   "zero_optimization": {
     "stage": 2,
     "offload_optimizer": {
-      "device": "cpu"
+        "device": "cpu"
     },
     "contiguous_gradients": true,
     "overlap_comm": true
   },
+  "optimizer": {
+    "type": "AdamW",
+    "params": {
+        "lr": "auto",
+        "betas": "auto",
+        "eps": "auto",
+        "weight_decay": "auto"
+    }
+},
```

Comment:
Would you be able to remove the optimizer from the file? It isn't compatible with transformers 4.30, thanks! After this change, LGTM!